StripeConnect/StripeConnect/Source/EmbeddedComponentManager.swift

```diff
@@ -28,10 +28,14 @@ public class EmbeddedComponentManager {
     // content should load.
     var shouldLoadContent: Bool = true
 
+    /// Return URL used for FinancialConnections
+    let returnUrl: String
+
     /**
      Initializes a StripeConnect instance.
 
      - Parameters:
+       - returnUrl: A return URL that can be used to return to the embedded component
```

Comment:
This will trigger another API review

Reply:
Actually, we might not need to do this – it looks like the returnURL is optional and only used if want to enable the native banking app to oauth and return back to the app.

Context:
https://stripe.slack.com/archives/C02KW8G938W/p1729221773775259?thread_ts=1729217697.015449&cid=C02KW8G938W

Reply:
turns out we don't need a return url